src/main/kotlin/no/nav/helse/dokument/K9MellomlagringGateway.kt

```diff
@@ -90,18 +90,22 @@ class K9MellomlagringGateway(
     }
 
     internal suspend fun slettDokmenter(
-        urls: List<URI>,
+        vedleggId: List<String>,
         dokumentEier: DokumentEier,
         correlationId: CorrelationId
     ) {
         val authorizationHeader = cachedAccessTokenClient.getAccessToken(k9MellomlagringScopes).asAuthoriationHeader()
 
         coroutineScope {
             val deferred = mutableListOf<Deferred<Unit>>()
-            urls.forEach {
+            vedleggId.forEach { vedleggId ->
```

Comment:
Fint om du kaller det for dokumentId. Det er ikke bare vedlegg vi oppretter eller slettet :)